feat: Adding all validations of #414 (#500)
belastingdienst/opr-paas: docs/administrators-guide/validations.md

# Introduction

Most fields in the CRD are checked directly with kubebuilder validations,

> But some options can be validated with a regular expression that can be set in PaasConfig:

Reviewer: ```suggestion
But some fields can be validated with a regular expression that can is configurable through the PaasConfig:
```

> Below snippet shows all options for validation and how to validate them:

Reviewer: ```suggestion
Below snippet shows the complete available validations which can be enabled. It also shows how validation is expected to be configured:
```
